README.md

```diff
@@ -320,14 +327,102 @@ const MapContainer = ({dispatch}) => (
 ```
 Read more about [forward dispatching actions][forward-actions]
 
+### 4. Customize style.
+Kepler.gl implements css styling using [Styled-Components](https://www.styled-components.com/). By using said framework Kepler.gl offers the ability to customize its style/theme using the following approaches:
+- Passing a Theme prop
+- Styled-Components ThemeProvider
+
+
+The available properties to customize are listed here [theme](https://github.com/uber/kepler.gl/blob/master/src/styles/base.js).
+
+[Custom theme example](https://github.com/uber/kepler.gl/tree/master/examples/custom-theme).
+
+#### Passing a Theme prop.
+You can customize Kepler.gl theme by passing a __theme__ props to Kepler.gl react component as it follows:
+```javascript
+// Properties are defined in https://github.com/uber/kepler.gl/blob/master/src/styles/base.js
+const white = '#ffffff';
+const customTheme = {
```

Comment:
nitpick, as this is a code example, we don't have to list 20+ custom themes here, 2-3 would be enough, for easy understanding

Reply:
sounds good